pkg/apis/servicecatalog/v1alpha1/types.go

```diff
@@ -223,11 +223,11 @@ type ServiceClass struct {
 
 // ServiceClassSpec represents details about the ServicePlan
 type ServiceClassSpec struct {
-	// ServiceBrokerName is the reference to the Broker that provides this
+	// ClusterServiceBrokerName is the reference to the Broker that provides this
 	// ServiceClass.
 	//
 	// Immutable.
-	ServiceBrokerName string `json:"brokerName"`
+	ClusterServiceBrokerName string `json:"brokerName"`
```

Comment:
JSON tag for this field should be `clusterServiceBrokerName`.

Reply:
done